File: 389/main.go
package main

import "fmt"


func main() {
	fmt.Println(findTheDifference("abc","abcd"))
}

func findTheDifference(s string, t string) byte {
	sumS:=0
	sumT:=0
	for _, v := range s {
		sumS+=int(v)
	}
	for _, v := range t {
		sumT+=int(v)
	}
	return byte(sumT-sumS)
}

File: 389/main.py
class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        sumS = 0
        sumT = 0
        for v in s:
            sumS += ord(v)
        for v in t:
            sumT += ord(v)
        return chr(sumT - sumS)